.gitignore

```diff
@@ -27,6 +27,7 @@ tools/clroot/db.sqlite3-wal
 .idea
 .vscode/
 *.iml
+yarn.lock
```

Comment:
Don't we commit this intentionally?

Reply:
There aren't any currently checked in, looks like the only one we had was intentionally removed on Oct 18th:
[3cd64d578361db491592a50947d0470ddefba6e9](https://github.com/smartcontractkit/chainlink/commit/3cd64d578361db491592a50947d0470ddefba6e9)

I added that because both that one and one under `./core/gethwrappers` get generated when you compile the solidity wrappers.  Then they show up as new files that git sometimes tries to stage for checking in.  From what I can tell, they appear to be mostly empty files, I'm not sure why because they used to contain more.

Reply:
If a file is generated then IMHO it should not be gitignored - if it starts to populate in the future it would be easy to overlook and would be left out until we did. What is the problem with a blank file?

Reply:
I just figured since it was intentionally removed from the repo (with comment "removing yarn.lock") there's probably a good reason why someone removed it, so I shouldn't add it back.

I don't care one way or another whether we add it to the repo or not, I just don't want extra files in my local checkout that I have to worry will accidentally get checked in.  In general, I think anything not in the repo should be ignored, right?

Reply:
@HenryNguyen5 can you provide any context on this?

Reply:
We may have removed the operator ui yarn lock file, but that doesn't have anything to do with generated gethwrappers. If a generation step is generating files, then they should be checked in, or the generation code should be modified to not produce those files.

Reply:
We dont use `yarn` anymore, we use `pnpm` now. That lockfile is being generated from https://github.com/smartcontractkit/chainlink/pull/7644/files#diff-d17e85b5b422ab0edd42791c7df53c6fc6bb26d3bd7ef279b5d890682e9a612aR12

After removal, you wont get any lockfiles generated.

Reply:
Related: https://github.com/smartcontractkit/chainlink/pull/7775